### PR TITLE
reor 0.1.46 (new cask)

### PR DESCRIPTION
**Pull request by chenrui333**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/r/reor.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      verified: "github.com/reorproject/reor/"
+  name "Reor"
+  desc "AI note-taking app that runs models locally"
+  homepage "https://reorproject.org/"
+
+  app "Reor.app"
+
+  zap trash: [
+    "~/Library/Application Support/Reor",
+    "~/Library/Preferences/YourAppID.plist",
```
Comment (bevanjkay) [inline Casks/r/reor.rb:18]: Is this definitely correct?

Comment (chenrui333) [reply Casks/r/reor.rb:18]: yeah, i just submitted the PR for it. (it is correct)

**Comment by chenrui333** [comment]
right now it failed with the notarization process, do we have any docs for it?

**Comment by bevanjkay** [comment]
The app isn't signed @chenrui333

**Comment by bevanjkay** [comment]
We only allow apps to be added if they are signed and notarized. You'll have to ask upstream to sign it before it can be included.

**Comment by chenrui333** [comment]
Make sense. Gonna close it for now.

**Comment by chenrui333** [comment]
- https://github.com/reorproject/reor/issues/25

**Pull request closed without merging**